### PR TITLE
Add Dandelion Sprout malware and some lil lists

### PR DESCRIPTION
**Pull request by GiddyGoatGaming**
im surprised dandelion sprout's malware list has not been added yet so I'm proposing adding that. Additionally, nextdns has some parental filters that are not on here so maybe add those also. they are extremely tiny.

### CODE REVIEW

File: edit_here_to_add_blocklists.json [hunk 1]
```diff
@@ -1127,5 +1127,50 @@
...
+        "category": "Malware",
+        "source": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/Alternate%20versions%20Anti-Malware%20List/AntiMalwareHosts.txt",
+        "totalDomains": 0,
+        "lastUpdated": "",
+        "value": 114
+    },
+    {
+        "loc":"custom_filters/normal/id115.txt",
+        "name": "nextdns (signal)",
+        "category": "Wildcard, Parentalcontrol",
```
Comment (bhanupratapys) [inline edit_here_to_add_blocklists.json:1143]: For wildcard filters, it should be like `"loc":"custom_filters/`**`wildcard`**`/idxxx.txt"`,

File: edit_here_to_add_blocklists.json [hunk 2]
```diff
@@ -1127,5 +1127,50 @@
...
+        "value": 117
+    },
+    {
+        "loc":"custom_filters/normal/id118xt",
+        "name": "nextdns (hbomax)",
+        "category": "Wildcard, Parentalcontrol",
+        "source": "https://raw.githubusercontent.com/nextdns/metadata/master/parentalcontrol/services/hbomax",
+        "totalDomains": 0,
+        "lastUpdated": "",
+        "value": 118
```
Comment (bhanupratapys) [inline edit_here_to_add_blocklists.json:1174]: I looked into these nextdns filter lists, they all have just 1 or 2 domains. Which wouldn't be ideal to have them in this setup. I'll create a separate list named as `dnswarden (blacklist wildcard extreme)` . So could you please remove these nextdns lists?

**Review by bhanupratapys — COMMENTED** [review]
Suggested some minor changes

File: edit_here_to_add_blocklists.json (hunk 1 above)
Comment (GiddyGoatGaming) [reply edit_here_to_add_blocklists.json:1143]: Yep, my bad. I forgot about that part😂. Il get that fixed asap

File: edit_here_to_add_blocklists.json (hunk 2 above)
Comment (GiddyGoatGaming) [reply edit_here_to_add_blocklists.json:1174]: Yep

**Pull request merged**